adapters/appnexus/appnexus.go

```diff
@@ -248,8 +249,17 @@ func (a *AppNexusAdapter) MakeRequests(request *openrtb.BidRequest) ([]*adapters
 	memberIds := make(map[string]bool)
 	errs := make([]error, 0, len(request.Imp))
 
+	// AppNexus openrtb2 endpoint expects imp.displaymanagerver to be populated, but some SDKs will put it in imp.ext.prebid instead
+	var defaultDisplayManagerVer string
+	if request.App != nil {
+		source, err1 := jsonparser.GetString(request.App.Ext, "prebid", "source")
+		version, err2 := jsonparser.GetString(request.App.Ext, "prebid", "version")
+		if (err1 == nil) && (err2 == nil) {
+			defaultDisplayManagerVer = fmt.Sprintf("%s-%s", source, version)
```

Comment:
FYI I actually profiled this a few months ago, and `source + "-" + version` runs much faster. Not a huge deal, but... *shrugs*